platform/src/main/java/org/stellar/anchor/platform/configurator/SpringFrameworkConfigurator.java

```diff
@@ -11,6 +13,16 @@ public class SpringFrameworkConfigurator extends AbstractConfigurator
   public void initialize(@NotNull ConfigurableApplicationContext applicationContext) {
     // Load and add the data access settings to Spring `Environment`
     PropertiesPropertySource pps = createPrefixedPropertySource("spring");
+    pps.getSource()
+        .forEach(
+            (String cfgName, Object cfgValue) -> {
+              if (cfgName.startsWith("logging.level") && cfgValue instanceof String) {
+                LoggingSystem system =
```

Comment:
nit: Do we need to call this for every source?

Reply:
Actually not, but it's just for one partner in the legacy branch (and most likely they will only use it for debug logs), so I decided to keep it simple